packages/components/phone-number/src/components/osds-phone-number/constants/phone-number-countries.ts

```diff
@@ -0,0 +1,7 @@
+enum ODS_PHONE_NUMBER_COUTRIE {
```

Comment:
`ODS_PHONE_NUMBER_COUNTRY`
maybe `ODS_PHONE_NUMBER_COUNTRY_PRESET` would be even better